kolena/_utils/state.py

```diff
@@ -168,14 +168,19 @@ def get_token(
 
 
 @contextlib.contextmanager
-def kolena_session(api_token: str, base_url: Optional[str] = None) -> Iterator[_ClientState]:
+def kolena_session(
+    api_token: str,
+    base_url: Optional[str] = None,
+    additional_request_headers: Optional[Dict[str, Any]] = None,
+) -> Iterator[_ClientState]:
     base_url = base_url or _get_api_base_url()
     init_response = get_token(api_token, base_url)
     client_state = _ClientState(
         base_url=base_url,
         api_token=api_token,
         jwt_token=init_response.access_token,
         tenant=init_response.tenant,
+        additional_request_headers=additional_request_headers,
```

Comment:
Can you check if we should also include `proxies`, which was previously missed, even though likely this method is currently used internally?